dev-client/tsconfig.json

```diff
@@ -9,5 +9,10 @@
     }
   },
   "include": ["src", "__tests__", "jest"],
-  "exclude": ["dist"]
+  "exclude": ["dist"],
+  "ts-node": {
+    "compilerOptions": {
+      "module": "commonjs",
```

Comment:
question: can we use ESM here? we will need to use some ESM dependencies.

Reply:
i don't know, this is a dev only config thing for the `transformNativeBase` script i explained below so i just did whatever made it work and then moved on